### ISSUE TITLE
5th Anniversary publication

### ISSUE THREAD

**Issue by NoushinN** [hackseq/hackseqRNA#22]
Hi everyone, as part of the plan to celebrate our 5th year anniversary, we are aiming for the publication of our 5-year retrospective learnings. [This](https://f1000research.com/articles/6-197) was the first one!
In this spirit, we have a pre-hackathon and post-hackathon survey to gauge attendance expectations.
[Here](https://docs.google.com/forms/u/3/d/1ElDiRj1NLGgOUP1lmb_AuEWGnBrPAjBLMzcNxB5c7Og/edit?usp=drive_web) is an attempt at the pre-hackathon survey for hackseqRNA; the questions are made to reflect what was first asked in the first publication, can you please have a look and make suggestions?

I am going to start a draft for publication sections, please let me know if you'd like to contribute and we will make a working group.
Thanks!

**Comment by NoushinN** [comment]
A draft publication skeleton is crafted (in hackseq account), can send invitation to working group: @ababaian could you discuss this in the next meeting? i.e. who's interested and can provide support/capacity? 
Other needs: longitudinal raw data collected in pre/post hackathons since 2016 so we can amalgamate

**Comment by NoushinN** [comment]
going through the deep dungeons of hackseq google drive, where can the results of the surveys be found? historically speaking, have we saved the responses throughout time?

**Comment by ababaian** [comment]
They are there somewhere, it was all google forms if that helps

**Comment by NoushinN** [comment]
Yes! got it...thinking of linking them all into one if possible to analyze, will update

**Comment by NoushinN** [comment]
Hey org committee, anyone up for reviving this task?
